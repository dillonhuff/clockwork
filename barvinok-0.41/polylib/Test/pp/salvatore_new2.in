3 5
#   x    y   a   cte
1   1    0  -1    0 # x >= a
1  -1    0   1    1 # x <= a+1
1   1    0   0    0 # x >= 0


0 3

a
