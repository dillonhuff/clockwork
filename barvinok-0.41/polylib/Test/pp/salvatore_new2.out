---------------------------------------
Overall rays :
Line:   [   0    1  ]
---------------------------------------
Domain :
         - a  >= 0
         a + 1 >= 0

Vertices :
[ 0,  0 ]
[ a+1,  0 ]
---------------------------------------
Domain :
         a  >= 0
          1 >= 0

Vertices :
[ a+1,  0 ]
[ a,  0 ]
